packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -774,10 +774,11 @@ test.describe('Routing', () => {
 	});
 
 	test('does not normalize external path', async ({ page }) => {
+		/** @type {string[] } */
 		const urls = [];
 
 		const { port, close } = await start_server((req, res) => {
-			if (req.url !== '/favicon.ico') urls.push(req.url);
+			if (req.url && req.url !== '/favicon.ico') urls.push(req.url);
```

Comment:
why do we need this? i don't believe `req.url` can ever be undefined, but even if it was then `undefined !== '/favicon.ico` is perfectly fine

Reply:
it was to make `urls` have type `string[]`. alternatively, we could change `urls` to `Array<string | undefined>`

Reply:
I made that change to use `Array<string | undefined>` instead